.circleci/config.yml

```diff
@@ -124,7 +127,10 @@ jobs:
                 name: Build << parameters.app >> Zalpha
                 command: |
                   if [[ ${APP_VERSION} == *"-rc-"* ]]; then
-                    bundle exec fastlane build_alpha app:<< parameters.app >> skip_confirm:true skip_prechecks:true create_release:true upload_to_play_store:true
+                    bundle exec fastlane build_alpha app:<< parameters.app >> skip_confirm:true skip_prechecks:true upload_to_play_store:true
+                  else
+                    echo "Skipping and halting this job because we only build Zalpha variants for rc versions, but current version is ${APP_VERSION}"
+                    circleci-agent step halt
```

Comment:
I needed to add that change because without it, this build step will be skipped as expected… but the rest of the job would continue, and ultimately fail on the `persist_to_workspace` step later in that job, as the `build/` directory would not exist. This is what happened in [this CircleCI run](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/26698/workflows/00952690-5b37-435d-9620-35e649b81c5a/jobs/115514).

So instead, this halts the job gracefully, and does not run the following steps after this one for the job, avoiding to try running the `persist_to_worspace` one.

I didn't encounter this during my earlier tests when I did the intermediate betas because we only skip the Zalpha during the _final_ release.